Directory.Build.props

```diff
@@ -7,6 +7,12 @@
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
   </PropertyGroup>
 
+  <PropertyGroup>
```

Comment:
The "Architecture" logic is handled in Arcade; defaults, et al - I don't feel that this is the correct place for this, or if it should be included at all.

Reply:
It doesn't look to me like Arcade handles this. (This is being discussed at https://github.com/dotnet/arcade/issues/4209.)

Reply:
Line 12 introduces a new paradigm; to wit: if the user has not defined an architecture parameter, then calculate the architecture. But why would we make the user explicitly define /architecture="x86" but not require the same treatment for "amd64"? I think it would be ok to ask the user to explicitly define the architecture if it is not 'x64', as this is what is happening now. However, I'm open to a calculation of the architecture.

Reply:
I never quite understood why we require developers to specify any architecture at all, instead of inferring one by default. Seems like it's just making the default workflow complicated.

@johnbeisner would you like me to try and handle x86 here too?

Reply:
> why would we make the user explicitly define /architecture="x86" but not require the same treatment for "amd64"?

Those archs are special because a win-x64 machine can also build win-x86. This code's arch detection then can depends on what process the build is running under, in particular with Visual Studio. This makes sense to some degree (maybe you foresaw this and you're ok with it for Core-SDK?) but is a definite change. This difference caused concrete issues in CoreFX leading to the idea of always defaulting to the detected arch being reverted: https://github.com/dotnet/corefx/pull/40427.

I believe it makes sense to go with the same pragmatic end result: only automatically handle arm64 for now to avoid getting into unintended messes like the above. Improving the defaults for non-arm64 archs can be tracked for later IMO, if it is desired.

Reply:
Thank you all for the discussion.